cfg/decode_hook.go

```diff
@@ -46,8 +49,11 @@ func stringToURLHookFunc() mapstructure.DecodeHookFuncType {
 func DecodeHook() mapstructure.DecodeHookFunc {
 	return mapstructure.ComposeDecodeHookFunc(
 		mapstructure.TextUnmarshallerHookFunc(),
-		stringToURLHookFunc(),
 		mapstructure.StringToTimeDurationHookFunc(), // default hook
 		mapstructure.StringToSliceHookFunc(","),     // default hook
+		// TODO: stringToURLHookFunc() needs to be in the end in order to avoid
+		// error being thrown.
```

Comment:
Maybe I understood this incorrectly but I don't think ordering will effect the error being thrown? Can you confirm?